src/VisualStudio/Core/Def/Venus/VenusCommandFilter.cs

```diff
@@ -71,18 +71,18 @@ protected override int GetDataTipTextImpl(TextSpan[] pSpan, out string pbstrText
 
                 // Next, we'll check to see if there is actually a DataTip for this candidate.
                 // If there is, we'll map this span back to the DataBuffer and return it.
-                pSpan[0] = candidateSpan.ToVsTextSpan();
```

Comment:
problem was here.  we were placing a span in our subject-buffer coordinates into an in/out parameter that is referring to the surface buffer.  If we ended up succeeding below, this wasn't a problem, as we'd overwrite this.  But if we failed, we'd leave this bogus data in here.